.circleci/config.yml

```diff
@@ -487,6 +503,40 @@ jobs:
           replace: $SNAPSHOT_VERSION
       - notify_github:
           data: "$(jq -Rcs '{ body: . }' scripts/snapshot-comment.md)"
+  build_check_android:
+    working_directory: ~/project
+    executor:
+      name: android/android-machine
+      tag: '2024.01.1'
+    steps:
+      - advanced-checkout/shallow-checkout
+      - build_and_pack_sdk:
+          working_directory: ~/project/examples/hybrid
+      - run:
+          name: Build Android
+          command: |
+            cd examples/hybrid/android
+            ./gradlew assembleDebug
```

Comment:
Let's also make this job generic for different example apps and build the app in release mode as mentioned in the previous comment. Let's do the same for `build_check_ios` as well.